File: Python/392. Is Subsequence.py
class Solution:
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        cnt, i = 0, 0
        while cnt < len(s) and i < len(t):
            if s[cnt] == t[i]:
                cnt += 1
            i += 1
        return cnt == len(s)

File: C++/392. Is Subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int cnt = 0;
        for(int i=0;cnt<s.length(),i<t.length();i++)
        {
            if(s[cnt]==t[i])
                cnt++;
        }
        return cnt == s.length();
    }
};